contracts/test/StablecoinConverterTestInterface.sol

```diff
@@ -0,0 +1,13 @@
+// TokenStore stores Tokens for applications, which have discrete States increasing with time
+pragma solidity ^0.5.0;
+
+import "@gnosis.pm/mock-contract/contracts/MockContract.sol";
+import "../StablecoinConverter.sol";
+
+
+contract StablecoinConverterTestInterface is StablecoinConverter {
```

Comment:
I really don't like the concept of subclassing for testing purposes. Here you are merely overriding a variable and then testing that somewhere in the code we check for this variable.

However, this is not really testing that e.g. we increment the numbers of token whenever we add a token and thus feels a bit hacky for the sake of getting 100% coverage.

I'd suggest to take the numbers of tokens in the constructor and then writing a unit test that instantiates the class with a small number of maxTokens.

Long term (if we really care about the gas efficiency of a constant, we should probably adjust @bh2smith multicaller library to allow calling addToken with different addresses many times in one transaction and thus test the limit like this).

Reply:
Yeah, good call. Done